packages/core/src/components/carousel/Carousel.astro

```
@@ -0,0 +1,55 @@
+  data-opts={JSON.stringify(opts)}
+  data-auto-init={autoInit}
+  {...rest}
+>
+  <slot />
+</div>
+
+<script>
+  import { initCarousel } from "./index";
+
+  const setupCarousels = () => {
+    const carousels = document.querySelectorAll(".starwind-carousel") as NodeListOf<HTMLElement>;
```

ESLint

[error] 'NodeListOf' is not defined.

(no-undef)